### DISCUSSION TITLE
Runing Linkerd outside of containers

### DISCUSSION THREAD

**Discussion by Trajanv** [linkerd/linkerd2#11126] [General]
Hi, As i see, Linkerd is service mesh for Kubernetes. is possible to run Linkerd outside of containers (k8s)? For instance, All of my app-instances could be running on VMs. If so, how can i leverage Linkerd ? Please provide samples/references.

**Comment by mateiidavid** [comment]
@Trajanv currently Linkerd only works in Kubernetes. It's on the roadmap to expand to other (non-Kubernetes) workloads, but it's not as of yet a thing. Keep an eye out on the announcements. :)
